src/java.base/share/classes/java/text/ChoiceFormat.java

```diff
@@ -558,7 +558,20 @@ StringBuf format(double number, StringBuf toAppendTo,
     }
 
     /**
-     * Parses a Number from the input text.
+     * Parses a {@code Number} from the input text, the subtype of which is always
+     * {@code Double}. The value returned is the {@code limit} corresponding
+     * to the {@code format} that is the longest substring of the input text.
+     * Matching is done in ascending order, when multiple {@code formats} match
```

Comment:
Nit: {@code format}s

Reply:
Sounds good. Addressed the conformance issue possibility in the CSR.